app/informativedialog.h

```diff
@@ -0,0 +1,23 @@
+#ifndef INFORMATIVEDIALOG_H
+#define INFORMATIVEDIALOG_H
+
+#include <QDialog>
+
+namespace Ui
+{
+class InformativeDialog;
+}
+
+class InformativeDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit InformativeDialog(QString title, QString description = QString(), QWidget *parent = 0 );
+    ~InformativeDialog();
+
+private:
+    Ui::InformativeDialog *ui;
+};
```

Comment:
I see. In that case the question here is whether we want consistency within Pencil or within each user’s operating system. Personally I feel that OS-level consistency is much more important unless a completely custom UI such as Blender’s is used, since most users have much more experience with the conventions of their system rather than the conventions of Pencil. That’s why so many toolkits and systems have [Human](https://developer.apple.com/macos/human-interface-guidelines/overview/themes/) [Interface](https://community.kde.org/KDE_Visual_Design_Group/HIG) [Guidelines](https://developer.gnome.org/hig/stable/) [and](https://developer.microsoft.com/en-us/windows/apps/design) [such](http://www.s-and-b.ru/syshlp/motif_guide/MotifStyleGuide/TOC.html).  In general I figure that Qt already does a rather decent job at consistency, there already are “standard buttons, toggles and such” and there are native styles for Windows and macOS as well as a GTK+ style for Linux. That’s just my two cents anyway.